pkg/controller/controller_instance.go

```diff
@@ -273,21 +273,6 @@ func (c *controller) reconcileServiceInstanceDelete(instance *v1alpha1.ServiceIn
 		request.OriginatingIdentity = originatingIdentity
 	}
 
-	// If the instance is failed, just clear the finalizer
-	if isServiceInstanceFailed(instance) {
-		glog.V(5).Infof("Clearing catalog finalizer from ServiceInstance %v/%v", instance.Namespace, instance.Name)
-
-		// Clear the finalizer
-		finalizers.Delete(v1alpha1.FinalizerServiceCatalog)
-		toUpdate.Finalizers = finalizers.List()
-
-		if _, err = c.updateServiceInstanceStatus(toUpdate); err != nil {
-			return err
-		}
-
-		return nil
-	}
-
 	// it is arguable we should perform an extract-method refactor on this
 	// code block
 
```

Comment:
If we are not going to clear out `CurrentOperation` when we start orphan mitigation, then we need to add logic here to set `OperationStartTime` when sending the first DeprovisionRequest for an orphan mitigation.